pkg/api/mlflow/dao/repositories/experiment.go

```diff
@@ -107,3 +108,31 @@ func (r ExperimentRepository) Update(ctx context.Context, experiment *models.Exp
 
 	return nil
 }
+
+// Delete removes the existing models.Experiment from the db.
+func (r ExperimentRepository) Delete(ctx context.Context, experiment *models.Experiment) error {
```

Comment:
I think these two signatures need to be added to ExperimentRepositoryProvider interface

Reply:
Fixed